test/TestServerProjectsLowLevel/dpg_update1/Custom/ParamsClient.cs

```diff
@@ -0,0 +1,184 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+
+namespace dpg_update1_LowLevel
+{
+    /// <summary> The Params service client. </summary>
+    [CodeGenSuppress("PostParametersAsync", typeof(RequestContent), typeof(RequestContext))]
```

Comment:
Curious: What do the [CodeGenSuppress] attributes do?

Reply:
The autogenerated `PostParametersAsync` has the issue of generating duplicate methods, so here we can use `CodeGenSuppress` to suppress the autogenerated `PostParametersAsync` methods whose input parameters are of the types of RequestContent and RequestContext. Then we can write the correct customized code for a `PostParametersAsync` method.

The generator will scan existing customized code (all the files in the project that are not under the Generated folder) for the attributes like CodeGenSuppress, then apply the actions like removing the method to the generated code with Roslyn code analyzer.

Reply:
I see, thank you for that explanation!

Is that the final approach we decided for addressing this problem, that we'll recommend not using the generated method and writing the two method overloads for different content types by hand?

Reply:
Per our offline conversation, we will handle this as part of a separate PR so we don't block this PR on that work.